pkg/interpolator/agentpool/params.go

```diff
@@ -0,0 +1,87 @@
+	properties := agentPool.Properties
+	location := agentPool.Location
+	parametersMap := map[string]interface{}{}
+	acsengine.AddValue(parametersMap, "location", location)
+	//acsengine.AddValue(parametersMap, "targetEnvironment", acsengine.GetCloudTargetEnv(location))
+	acsengine.AddValue(parametersMap, "linuxAdminUsername", properties.LinuxProfile.AdminUsername)
+	acsengine.AddValue(parametersMap, "sshRSAPublicKey", properties.LinuxProfile.SSH.PublicKeys[0].KeyData)
+	for i, s := range properties.LinuxProfile.Secrets {
+		acsengine.AddValue(parametersMap, fmt.Sprintf("linuxKeyVaultID%d", i), s.SourceVault.ID)
+		for j, c := range s.VaultCertificates {
+			acsengine.AddValue(parametersMap, fmt.Sprintf("linuxKeyVaultID%dCertificateURL%d", i, j), c.CertificateURL)
+		}
+	}
+	//var KubernetesVersion api.OrchestratorVersion
+	KubernetesVersion := api.OrchestratorVersion(properties.KubernetesVersion)
+
+	cloudSpecConfig := acsengine.GetCloudSpecConfig(location)
+
+	// Agentpool parameters
+	for _, agentProfile := range properties.AgentPoolProfiles {
+		acsengine.AddValue(parametersMap, fmt.Sprintf("%sCount", agentProfile.Name), agentProfile.Count)
+		acsengine.AddValue(parametersMap, fmt.Sprintf("%sVMSize", agentProfile.Name), agentProfile.VMSize)
+	}
+
+	// Jumpbox
+	acsengine.AddValue(parametersMap, "jumpboxVmSize", properties.JumpBoxProfile.VMSize)
+	acsengine.AddValue(parametersMap, "jumpboxCount", properties.JumpBoxProfile.Count)
+	acsengine.AddValue(parametersMap, "jumpboxEndpointDNSNamePrefix", properties.DnsPrefix)
+
+	// Certificate information
+	acsengine.AddSecret(parametersMap, "apiServerCertificate", properties.CertificateProfile.APIServerCertificate, true)
+	acsengine.AddSecret(parametersMap, "apiServerPrivateKey", properties.CertificateProfile.APIServerPrivateKey, true)
+	acsengine.AddSecret(parametersMap, "caCertificate", properties.CertificateProfile.CaCertificate, true)
+	acsengine.AddSecret(parametersMap, "caPrivateKey", properties.CertificateProfile.CaPrivateKey, true)
+	acsengine.AddSecret(parametersMap, "clientCertificate", properties.CertificateProfile.ClientCertificate, true)
+	acsengine.AddSecret(parametersMap, "clientPrivateKey", properties.CertificateProfile.ClientPrivateKey, true)
+	acsengine.AddSecret(parametersMap, "kubeConfigCertificate", properties.CertificateProfile.KubeConfigCertificate, true)
+	acsengine.AddSecret(parametersMap, "kubeConfigPrivateKey", properties.CertificateProfile.KubeConfigPrivateKey, true)
+
+	acsengine.AddValue(parametersMap, "dockerEngineDownloadRepo", cloudSpecConfig.DockerSpecConfig.DockerEngineRepo)
+	acsengine.AddValue(parametersMap, "kubernetesHyperkubeSpec", KubernetesImagebase+acsengine.KubeImages[KubernetesVersion]["hyperkube"])
+	//acsengine.AddValue(parametersMap, "kubernetesAddonManagerSpec", cloudSpecConfig.KubernetesSpecConfig.KubernetesImageBase+acsengine.KubeImages[KubernetesVersion]["addonmanager"])
```

Comment:
They were handy to have while working on pulling agent pools params out into their own package... if I hit a bug where some data was missing I could usually just find it here, and uncomment and get it working again..

Kind of a minimalistic approach.. include the bare minimum instead of all of them (but leave the others around as we're developing)

I can take them out